### ISSUE TITLE
Не работает автодополнение команд

### ISSUE THREAD

**Issue by ziceptor** [GreWeMa/gwm_Crates#9]
Это делает работу очень не удобной.

**Comment by iLefty** [comment]
Он имеет ввиду, когда пишешь команду и жмёшь таб, то команда сама вписывается в чат.

**Comment by GreWeMa** [comment]
Да, это я понял, надо будет просто переписать комманды по нормальному, а не как сейчас.
Когда-то -- займусь!

**Comment by ziceptor** [comment]
Не просто команда вписывается в чат, а так же аргументы.
